src/test/scala/com/mpc/scalats/core/ScalaParserSpec.scala

```diff
@@ -1,21 +1,23 @@
 package com.mpc.scalats.core
 
-import scala.collection.immutable.ListSet
-
-import org.scalatest.{ FlatSpec, Matchers }
-
-import ScalaModel._
+import com.mpc.scalats.core.ScalaModel._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.slf4j.LoggerFactory
 
+import scala.collection.immutable.ListSet
 import scala.reflect.runtime.universe.runtimeMirror
 
 /**
  * Created by Milosz on 06.12.2016.
  */
-final class ScalaParserSpec extends FlatSpec with Matchers {
+final class ScalaParserSpec extends AnyFlatSpec with Matchers {
```

Comment:
Why?

Reply:
Due to version bump of scalatest to 3.10.0